docs/New-ExternalHelp.md

```diff
@@ -5,82 +5,74 @@ schema: 2.0.0
 
 
 Mode                LastWriteTime         Length Name
 ----                -------------         ------ ----
 -a----        5/19/2016  12:32 PM          46776 platyPS-help.xml
 ```
 
-Create external help file in output path directory.
-Note that directory should include language name.
-### Example 1 (With -Force and custom encoding)
+This command creates an external help file in the specified location. This command uses the best practice that the folder name includes the locale.
+
+### Example 2: Create help that uses custom encoding
 ```
-PS C:\> New-ExternalHelp .\docs -OutputPath out\platyPS\en-US -Force -Encoding ([System.Text.Encoding]::Unicode)
+PS C:\> New-ExternalHelp -Path ".\docs" -OutputPath "out\PlatyPS\en-US" -Force -Encoding ([System.Text.Encoding]::Unicode)
 
 
-    Directory: D:\dev\platyPS\out\platyPS\en-US
+    Directory: D:\Working\PlatyPS\out\PlatyPS\en-US
 
 
 Mode                LastWriteTime         Length Name
 ----                -------------         ------ ----
 -a----        5/22/2016   6:34 PM         132942 platyPS-help.xml
 ```
 
-Create and overwrite existing external help file in output path directory.
-Use Unicode Encoding for output file.
+This command creates an external help file in the specified location. This command specifies the *Force* parameter, therefore, it overwrites an existing file. The command specifies Unicode encoding for the created file.
 ## PARAMETERS
 
 ### -OutputPath
-Path to a folder where you want to put your external help file(s).
-The name should end with a locale folder, i.e. ".\out\platyPS\en-US".
+Specifies the path of a folder where this cmdlet saves your external help file. The folder name should end with a locale folder, as in the following example: ```.\out\PlatyPS\en-US\```.
 
 
 ```yaml
 Type: String
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -Encoding
-Character encoding for your external help file.
-
-It should be of the type \[System.Text.Encoding\].
-You can control [precise details](https://msdn.microsoft.com/en-us/library/ms404377.aspx) about your encoding.
-For [example](http://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom), 
-you can control BOM (Byte Order Mark) preferences with it.
+Specifies the character encoding for your external help file. Specify a **System.Text.Encoding** object. For more information, see [Character Encoding in the .NET Framework](https://msdn.microsoft.com/en-us/library/ms404377.aspx) in the Microsoft Developer Network. For example, you can control Byte Order Mark (BOM) preferences. For more information, see [Using PowerShell to write a file in UTF-8 without the BOM](http://stackoverflow.com/questions/5596982/using-powershell-to-write-a-file-in-utf-8-without-the-bom) at the Stack Overflow community.
```

Comment:
Consider use one sentence per line in markdown

http://rhodesmill.org/brandon/2012/one-sentence-per-line/